src/SignalRServiceExtension/Bindings/SignalRAsyncCollector.cs

```diff
@@ -43,7 +43,11 @@ public async Task AddAsync(T item, CancellationToken cancellationToken = default
                     throw new ArgumentException("GroupName and UserId can not be specified at the same time.");
                 }
```

Comment:
since there's one more parameter added and have priority order, either do the check more accurately or leave it with priority logic in below but give users guidance in some docs.